docs/pages/server-access/guides/teleport-ssh.mdx

```diff
@@ -0,0 +1,241 @@
+      --ca-pin=sha256:c75f8be5b21a5cc1b6f91a71200f30a9a083dfea4fc47ec1c55143ccce8882e4 \
+      --auth-server=172.31.17.124:3025
+   ```
+
+   <Admonition type="warning" title="Warning">
+      Remember to replace the IP address supplied by default above with the `auth.example.com` proxy address: `auth.example.com:443`.
+   </Admonition>
+
+3. Run the command above with the correct `auth.example.com` public proxy address on the machine you wish to connect.
+
+4. You should now be able to view both Nodes at: `https://auth.example.com/web/cluster/auth.example.com/nodes` in the Web Interface after logging in with your user `auth-admin`.
+
+   ![Both Nodes in](../../../img/server-access/teleport_ui.png)
+
+## Connect to a cluster using tsh ssh without OpenSSH
+
+Teleport SSH is robust enough that you can do all the things you'd do with OpenSSH, without it.
+
+To connect to a cluster (`auth.example.com`) with public proxy `auth.example.com:443` as user `auth-admin` using `tsh ssh`:
+
+1. `tsh login --proxy=auth.example.com:443 --user=auth-admin`  
+
+2. You'll be prompted to for the password and One Time Passcode we set up previously.
+
+3. Your user will now see:
+
+   ```bash
+   Profile URL:        https://auth.example.com:443
+     Logged in as:       auth-admin
+     Cluster:            auth.example.com
+     Roles:              access, admin, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         disabled
+     Valid until:        2021-04-30 06:39:13 -0500 CDT [valid for 12h0m0s]
+     Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+   ```
+
+   `auth-admin` is now logged into the `auth.example.com` cluster and `auth` Node via Teleport SSH - no third-party tool required.
+
+4. `auth-admin` can now execute the following to find the cluster's `nodenames` which are used for direct SSH connections:
+
+   i. `tctl nodes ls`  
+   ii. The `auth` node is located on the bottom line below:
+
+   ```bash
+   Nodename         UUID                                 Address        Labels                                
+   ---------------- ------------------------------------ -------------- ------------------------------------- 
+   ip-172-31-30-155 8f1e2719-3161-400f-b664-3279cf046511                env=example,hostname=ip-172-31-30-155 
+   ip-172-31-17-124 e7719967-b6eb-4fc0-8e53-00fac6d896d4 127.0.0.1:3022 env=example,hostname=ip-172-31-17-124 
+   ```
+
+5. Using the command `tsh ssh root@ip-172-31-17-124` locally, `auth-admin` can SSH into the `auth` Node and do the following:
+
+   i. Connect to other Nodes in the cluster by repeating the steps above.  
+   ii. Traverse the Linux file system.  
+   iii. Execute desired commands.  
+
+   - The `tsh ssh` command allows one to do anything they would if they were to SSH into a server using a third-party tool. 
+   - All commands executed by `auth-admin` are recorded and can be replayed in the Teleport Web interface.
+
+## Configure Single Sign-On
```

Comment:
This should be a separate SSO for SSH guide. Will give you more space to talk about the SSO specific features in more detail.

I suggest:  remove the `teleport-ssh` guide out of scope of this PR, split this guide into SSO and Access requests guides.